server/index.js

```diff
@@ -24,6 +25,33 @@ const db = new pg.Pool({
   }
 });
 
+app.post('/api/egg', uploadsMiddleware, (req, res, next) => {
+  const message = 'message';
+  if (message in req.body) {
+    const { message, latitude, longitude, user } = req.body;
```

Comment:
You should get the user info from the `req.user` created by your authorization middleware.